haunt-roll-fail/arcs/game.scala

```diff
@@ -1490,15 +1490,16 @@ class Game(val setup : $[Faction], val options : $[Meta.O]) extends BaseGame wit
                 f.hand --> l --> deck
```

Comment:
I think this line might be an issue if it is executed twice. Cards from list `l` will no longer be in hand. Easiest work-around would be checking each card before moving:
```
l.foreach { d =>
    if (f.hand.has(d))
        d --> deck
}
```

Reply:
It must be my lack of understanding of the code. I thought if none of `l` is in hand, the line will not have any actual effect.